eox_nelp/course_experience/api/v1/tests/mixins_helpers.py

```diff
@@ -377,3 +380,316 @@ def test_post_not_found_course_id(self):
 
         self.assertIn(response.headers["Content-Type"], RESPONSE_CONTENT_TYPES)
         self.assertContains(response, "object does not exist.", status_code=status.HTTP_400_BAD_REQUEST)
+
+
+class PublicExperienceTestMixin:
+    """
+    A mixin class with base experience setup configuration for testing.
+    """
+
+    def setUp(self):  # pylint: disable=invalid-name
+        """
+        Set base variables and objects across experience test cases.
+        """
+        self.patchers = [
+            patch("eox_nelp.course_experience.api.v1.public_views.configuration_helpers"),
+        ]
+        self.configuration_helpers_mock = self.patchers[0].start()
+        self.configuration_helpers_mock.get_current_site_orgs.return_value = ["org1"]
+
+        self.client = APIClient()
+        self.users = User.objects.bulk_create(
+            [
+                User(username="luke", is_superuser=True, id=1000),
+                User(username="han", id=1001),
+                User(username="chewi", id=1002),
+                User(username="citripio", id=1003),
+                User(username="R2D2", id=1004),
+            ]
+        )
+        self.course_overviews = CourseOverview.objects.bulk_create(
+            [
+                CourseOverview(id="course-v1:org1+multiple+2023-t1", org="org1"),
+                CourseOverview(id="course-v1:org1+multiple+2023-t2", org="org1"),
+                CourseOverview(id="course-v1:org2+multiple2+2023-t2", org="org2"),
+            ]
+        )
+        self.client.force_authenticate(self.users[0])  # superuser default
+
+    def tearDown(self):  # pylint: disable=invalid-name
+        """Stop patching."""
+        for patcher in self.patchers:
+            patcher.stop()
+
+    def test_post_not_allowed(self):
+        """
+        Test disallow by credentials the  get  request to the list endpoint
+         for the desired view.
+        Expected behavior:
+            - Status code 405.
+        """
+        self.client.force_authenticate(user=None)
```

Comment:
is it necessary ?

Reply:
no [43d0537](https://github.com/eduNEXT/eox-nelp/pull/53/commits/43d05375ae2d16bb11a39c9e4d368b2c235ece72)